lib/resources/datasets.js

```diff
@@ -19,12 +19,12 @@ module.exports = (service, endpoint) => {
       .then((project) => auth.canOrReject('dataset.list', project))
       .then(() => Datasets.getAllByProjectId(params.id))));
 
-  service.get('/projects/:projectId/datasets/:name/download', endpoint(({ Datasets, Entities, Projects }, { params, auth }, _, response) =>
+  service.get('/projects/:projectId/datasets/:name.csv', endpoint(({ Datasets, Entities, Projects }, { params, auth }, _, response) =>
```

Comment:
I like the `:name.csv` endpoint, it works with integration tests that we have, do you see any problem if dataset name is strange? I can definitely change this to `entities.csv`